Kconfig

```diff
@@ -183,6 +183,13 @@ menu "LVGL configuration"
 				it is buffered into a "simple" layer before rendering. The widget can be buffered in smaller chunks.
 				"Transformed layers" (if `transform_angle/zoom` are set) use larger buffers and can't be drawn in chunks.
 
+		config LV_DRAW_THREAD_STACK_SIZE
+			int "Stack size of draw thread in bytes"
+			default 8192
+			depends on LV_USE_OS > 0
```

Comment:
It also depends on `LV_USE_DRAW_SW` as its the stack size of the SW render task.

Reply:
`LV_DRAW_THREAD_STACK_SIZE` should work as a general configuration, not just for software rendering.

Reply:
It's not necessarily true that all renderes need 8k RAM by default. A simpler one optimized of for size with less features might use only 2kB, but a heavier one might need much more.  

E.g. [PXP](https://github.com/lvgl/lvgl/blob/master/src/draw/nxp/pxp/lv_draw_pxp.c#L95) has a hard coded 2kB stack size. So `LV_DRAW_THREAD_STACK_SIZE` is not really a general configuration.

Reply:
Let's come to a conclusion on this before merging.

Reply:
PXP and other draw units should also use `LV_DRAW_THREAD_STACK_SIZE` as the stack size configuration instead of hard coding. If freetype rendering fonts are used in the PXP thread, a 2K stack is not enough.

Reply:
So there are 2 cases:
1. The GPU is limited and doesn't really depend on `lv_conf.h` :arrow_right: the stack size can be hardcoded (e.g. PXP)
2. The GPU is more powerful and can use FreeType or ThorVG if enabled :arrow_right:  it should use `LV_DRAW_THREAD_STACK_SIZE`.

If we approach it like this, I agree with current version. :+1: